specification/web/resource-manager/readme.md

```diff
@@ -464,3 +464,20 @@ java:
 regenerate-manager: true
 generate-interface: true
 ```
+
+### Tag: package-2018-03-01-hybrid

```

Comment:
title is not consistent with test later which uses `package-2018-03-01-hybrid-csharp`